src/Agent.Worker/TestResults/Legacy/LegacyTestRunDataPublisher.cs

```diff
@@ -35,15 +36,21 @@ public class LegacyTestRunDataPublisher : AgentService, ILegacyTestRunDataPublis
         private const string _testRunSystemCustomFieldName = "TestRunSystem";
         private readonly object _sync = new object();
         private int _runCounter = 0;
+        private IFeatureFlagService _featureFlagService;
+        private bool _calculateTestRunSummary;
+        private string _testRunner;
 
         public void InitializePublisher(IExecutionContext context, string projectName, VssConnection connection, string testRunner, bool publishRunLevelAttachments)
         {
             Trace.Entering();
             _executionContext = context;
             _projectName = projectName;
-            _resultReader = GetTestResultReader(testRunner, publishRunLevelAttachments);
+            _testRunner = testRunner;
+            _resultReader = GetTestResultReader(_testRunner, publishRunLevelAttachments);
             _testRunPublisher = HostContext.GetService<ITestRunPublisher>();
+            _featureFlagService = HostContext.GetService<IFeatureFlagService>();
             _testRunPublisher.InitializePublisher(_executionContext, connection, projectName, _resultReader);
+            _calculateTestRunSummary = _featureFlagService.GetFeatureFlagState("TestManagaement.PTR.GetTestRunSummary", new Guid("00025394-6065-48CA-87D9-7F5672854EF7"));
```

Comment:
Need to mock this for the l0

Reply:
yeah. not planning as of now, this is going to change again.